### DISCUSSION TITLE
Failure when building gnu.org/coreutils

### DISCUSSION THREAD

**Discussion by dio** [pkgxdev/pkgx#295] [Help]
We tried to understand the way we package stuff. Currently, we are interested in packaging gnu-sed. Since it is one of the things that we need to solve, i.e. issue introduced by: BSD vs GNU sed in most of our projects.

However, we caught by: `error: Uncaught (in promise) Error: too short to parse without a ``v`` prefix: 4.9`. And it is the same error when we try to build coreutils locally, below. Did we miss something?

```
   tea: pantry.core/scripts/build.ts gnu.org/coreutils
error: Uncaught (in promise) Error: too short to parse without a `v` prefix: 9.1
      if (!tolerant && parts.length < 3 && !vprefix) throw new Error(`too short to parse without a \`v\` prefix: ${input}`)
                                                           ^
    at new SemVer (https://raw.githubusercontent.com/teaxyz/cli/v0.18.1/src/utils/semver.ts:29:60)
    at https://raw.githubusercontent.com/teaxyz/cli/v0.18.1/src/hooks/usePantry.ts:271:32
    at Array.map (<anonymous>)
    at getVersions (https://raw.githubusercontent.com/teaxyz/cli/v0.18.1/src/hooks/usePantry.ts:271:21)
    at async Object.resolve (https://raw.githubusercontent.com/teaxyz/cli/v0.18.1/src/hooks/usePantry.ts:42:20)
    at async file:///Users/dio/src/tea/pantry.core/scripts/build.ts:45:15

```

Thanks!

**Comment by jhheider** [comment]
I believe that regression was introduced in https://github.com/teaxyz/cli/commit/082d800ab8bda5f8ae9b73b460e7738b9200688d. I likely need to audit both pantries for any 2-digit versions.

In the near term, you should be able to prefix their explicit version numbers with `v` to work around it.

Edit: I did notice the same when packaging `gcc`, and I used the above trick in the near term: https://github.com/teaxyz/pantry.extra/pull/97/files#diff-8d0ed87b60d5c3396f1a6e6a7403ec5623211264354c133c32750e0df5616cdf

**Comment by mxcl** [comment]
My bad, should be fixed: https://github.com/teaxyz/cli/pull/298

LMK if you need help with packaging!
